app/labor/badge_rewarder.rb

```diff
@@ -90,7 +97,7 @@ def self.award_streak_badge(num_weeks)
       count = 0
       num_weeks.times do |i|
         num = i + 1
-        count += 1 if user.articles.published.where("published_at > ? AND published_at < ?", num.weeks.ago, (num - 1).weeks.ago).any?
+        count += 1 if user.articles.published.where(published_a: num.weeks.ago..(num - 1).weeks.ago).any?
```

Comment:
Same as above, switched to a `BETWEEN` query. Same concerns apply /cc @benhalpern